paper/ref.bib

```diff
@@ -112,3 +113,29 @@ @article{ForetsS21
   url       = {https://doi.org/10.21105/jcon.00097},
   doi       = {10.21105/jcon.00097}
 }
+
+@book{neumaier1993wrapping,
+  title={The wrapping effect, ellipsoid arithmetic, stability and confidence regions},
+  author={Neumaier, Arnold},
+  year={1993},
+  publisher={Springer}
+}
+
+@article{karhbet2017range,
+  title={Range bounds of functions over simplices, for branch and bound algorithms},
+  author={Karhbet, Sam and Kearfott, Ralph Baker},
+  journal={Reliable Comput},
+  volume={25},
+  number={7},
+  pages={53--73},
+  year={2017}
+}
+
+@incollection{vzilinskas2007survey,
+  title={A survey of methods for the estimation ranges of functions using interval arithmetic},
+  author={{\v{Z}}ilinskas, Julius and Bogle, Ian David Lockhart},
+  booktitle={Models and Algorithms for Global Optimization: Essays Dedicated to Antanas {\v{Z}}ilinskas on the Occasion of His 60th Birthday},
+  pages={97--108},
+  year={2007},
+  publisher={Springer}
+}
```

Comment:
```suggestion
  publisher={Springer},
  doi={10.1007/978-0-387-36721-7\_6}
}

```

Reply:
is the backslash intentional?

Reply:
Yes, otherwise for me it did not work. But we may have to run the script to double-check.

Reply:
ok. anyways, once this is merged I'll run the `@whedon check references` on the review issues. We can then track down all possible DOIs issues